### PR TITLE
Fix sync status tests

### LINKED ISSUE #958: Fix test on develop
Looks like I've merged https://github.com/openmsupply/open-msupply/pull/802 with test error (cause by missing sync settings when initialisation status is initialise and username is required).

### PR DESCRIPTION
**Pull request by andreievg**
closes #958

### CODE REVIEW

File: server/cli/src/refresh_dates.rs [hunk 1]
```diff
@@ -45,8 +45,6 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("sync_log", "integration_started_datetime"),
         ("sync_log", "integration_finished_datetime"),
         ("activity_log", "datetime"),
-        ("location_movement", "enter_datetime"),
```
Comment (andreievg) [inline server/cli/src/refresh_dates.rs:48]: This is doubled up and causing postgres tests to fail

File: server/service/src/sync/sync_serde.rs [hunk 2]
```diff
@@ -59,9 +59,9 @@ where
 /// change on our side.
 pub fn naive_time<'de, D: Deserializer<'de>>(d: D) -> Result<NaiveTime, D::Error> {
     let secs = u32::deserialize(d)?;
-    /// using the _opt version of the method and on error returning a time of 00:00:00
-    /// as there have been some invalid time values returned by 4D - unsure of the origin of these
-    /// if the deserialisation panics then the whole server crashes, so have used the error & default
+    // using the _opt version of the method and on error returning a time of 00:00:00
```
Comment (andreievg) [inline server/service/src/sync/sync_serde.rs:62]: This was causing a warning `///` means doc comment (can't do those for expressions)

**Review by mark-prins — APPROVED** [review]
👍 LGTM!

**Comment by andreievg** [comment]
🙏 @mark-prins

**Pull request merged**